test/index.test.ts

```diff
@@ -49,6 +49,30 @@ describe('monika', () => {
     getPublicNetworkInfoStub.restore()
   })
 
+  test
+    .stderr()
+    .do(() => cmd.run(['--config', 'https://fake.url.com']))
```

Comment:
Please use https://example.com that is reserved for testing purposes. More info: https://www.rfc-editor.org/rfc/rfc2606.html.

Reply:
done... thanks